app/src/main/java/mollie/tictactoe/GameHelper.java

```diff
@@ -0,0 +1,58 @@
+package mollie.tictactoe;
+
+import mollie.tictactoe.players.MobilePlayer;
+import ttt.game.Board;
+import ttt.game.GameEngine;
+import ttt.game.Marks;
+
+public class GameHelper {
+
+    private GameEngine mGameEngine;
+
+    public GameHelper(GameEngine game) {
+        mGameEngine = game;
+    }
+
+    public GameHelper() {
+    }
+
+    public GameEngine createGame() {
+        mGameEngine = new GameEngine(new MobilePlayer(Marks.X), new MobilePlayer(Marks.O), new Board(3));
+        return mGameEngine;
+    }
+
+    public String playMove(int moveLocation) {
+        mGameEngine = getGame();
+        mGameEngine.play(moveLocation);
+        return mGameEngine.board(moveLocation).toString();
+    }
+
+    public GameEngine getGame() {
+        if (isNotStarted()) {
+            createGame();
+        }
+        return mGameEngine;
+    }
+
+    public Board getBoard() {
+        if (isNotStarted()) {
+            return new Board(3);
+        }
+        return mGameEngine.showBoard();
+    }
+
+    private boolean isNotStarted() {
+        return mGameEngine == null;
+    }
+
+    public boolean gameIsOver() {
+        return mGameEngine.isOver();
+    }
+
+    public String getWinner() {
+        if (mGameEngine.isWon()) {
+            return mGameEngine.winningMark().toString();
+        }
+        return null;
```

Comment:
@felipesere I appreciate this comment, and agree that an Optional would be much nicer than a null check, however because I am using Android API 23 Java 8 features are not supported.  I could switch to only target API 24 (Android 7) but that would be a conversation I would have to have with my client...